pkgs/by-name/rt/rtklib-explorer/package.nix

```
@@ -0,0 +1,45 @@
+{ stdenv
+, cmake
+, nix-update-script
+, blas
+, lapack
+, lib
+, fetchFromGitHub
+, qt6
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

When I read through https://github.com/NixOS/nixpkgs/issues/315337, it seems like there is no consensus of which method is the best here. My current opinion is that finalAttrs.version is a foot gun, which appears to update the git URL correctly to override the fetched version. But since the hash is not updated by only updating the version, the package will either fail to build due to a hash mismatch or silently use the old version if the user has already fetched the old version.

So my thinking is to keep the rec. But feel free to school me if you have any good arguments in this specific case.